### ISSUE TITLE
Cannot find module 'Glob'

### ISSUE THREAD

**Issue by ggrimbert** [cspotcode/vscode-mocha-latte#4]
Hi,

I am having this error when trying to run any test 

```
Error: Cannot find module 'Glob'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/ggrimbert/.vscode/extensions/cspotcode.vscode-mocha-latte-0.1.2/worker/findtests.js:4:25)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
```

**Comment by azusa0127** [comment]
Same issue here on Ubuntu.

**Comment by cspotcode** [comment]
@ggrimbert @azusa0127 Silly case insensitivity issue from before I forked the project.  Thanks for the report!

I just published 0.2.0 which should fix this bug.  Please let me know if you have any more problems by replying here or filing a new issue.  Thanks for you patience.
